cloudinit/util.py

```diff
@@ -520,13 +520,25 @@ def multi_log(text, console=True, stderr=True,
 
 @lru_cache()
 def is_BSD():
-    return 'BSD' in platform.system()
+    if 'BSD' in platform.system():
```

Comment:
how come we're not using `system_info()['variant']`

 it was most likely already called, so it's LRU cached at this point

Reply:
please disregard this… we might run into a dependency loop since `system_info()` calls `get_linux_distro()` which calls `is_BSD()`…